c10/util/Half.h

```diff
@@ -374,6 +374,8 @@ struct alignas(4) complex<Half> {
   }
   inline complex(c10::complex<float> value)
       : real_(value.real()), imag_(value.imag()) {}
+  inline complex(c10::complex<double> value)
```

Comment:
what is this change for?

Reply:
To support `torch::utils::store_scalar()` for c10::complex<Half>